syllabus/badges.md

```diff
@@ -296,3 +297,29 @@ style revision fill:#2cf
 style earned fill:#2cf
 
 ```
+
+## Community Badges
+
+These are the instructions from your `community_contributions.md` file: 
+For each one: 
+- add an item in a bulleted list (start the line with `- ` )
```

Comment:
- In the community_contributions.md file on your kwl repo, add an item in a bulleted list (start the line with - )